src/hotspot/cpu/x86/c2_MacroAssembler_x86.cpp

```diff
@@ -3901,6 +3941,10 @@ void C2_MacroAssembler::evmasked_op(int ideal_opc, BasicType eType, KRegister ma
       evpsrld(dst, mask, src1, src2, merge, vlen_enc, is_varshift); break;
     case Op_URShiftVL:
       evpsrlq(dst, mask, src1, src2, merge, vlen_enc, is_varshift); break;
+    case Op_RotateLeftV:
+      evrold(eType, dst, mask, src1, src2, merge, vlen_enc); break;
+    case Op_RotateRightV:
+      evrord(eType, dst, mask, src1, src2, merge, vlen_enc); break;
```

Comment:
is_varshift is not passed to evrold/evrord here. But x86.ad is sending is_varshift for rotates.